### DISCUSSION TITLE
How to list Sentry SDK for Android public vulnerabilities?

### DISCUSSION THREAD

**Discussion by daudrain** [getsentry/sentry-java#3587] [Q&A] [answered]
Hi,

The Security section of this repository doesn't list any security issues and searching for vulnerabilities in the [National Vulnerabilities Database](https://nvd.nist.gov/products/cpe/search) doesn't list any CVE related to Sentry SDK for Java/Android.

Only Python and next.js appear when searching with the following CPE code [cpe:2.3:a:sentry:sentry_software_development_kit:*:*:*:*:*:*:*:*](https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=cpe%3A2.3%3Aa%3Asentry%3Asentry_software_development_kit%3A*%3A*%3A*%3A*%3A*%3A*%3A*%3A*)

And searching with [cpe:2.3:a:sentry:sentry:*:*:*:*:*:*:*:*](https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=cpe%3A2.3%3Aa%3Asentry%3Asentry%3A*%3A*%3A*%3A*%3A*%3A*%3A*%3A*) lists vulnerability for the main sentry repository.

What would be the cpe code for Sentry SDK for Android?

Thank you

**Comment by markushi** [comment, marked as answer]
Hi @daudrain, thanks for reaching out!

As of now we haven't published any CVEs for the `sentry-java` SDK. We usually publish advisories on GitHub ([`sentry-java` link here](https://github.com/getsentry/sentry-java/security/advisories)) and request a CVE if necessary.

As there are no advisories for `sentry-java` right now, [here's an example](https://github.com/getsentry/sentry-javascript/security/advisories) how this looks like for our JS SDK.

**Comment by Harpoonthrower** [comment]
ok
